mobile/lib/src/infrastructure/settings_repository/settings_repository.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:polydodo/src/common/constants.dart';
+import 'package:polydodo/src/domain/settings/i_settings_repository.dart';
+import 'package:polydodo/src/domain/settings/settings.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class SettingsRepository extends ISettingsRepository {
+  SharedPreferences prefs;
+
+  SettingsRepository();
+
+  @override
+  Future<Map<String, dynamic>> getSettings() async {
+    prefs = (await SharedPreferences.getInstance());
+    return {
+      AGEKEY: prefs.getInt(AGEKEY),
+      BOARDKEY: AcquisitionBoard
+          .values[(prefs.getInt(BOARDKEY)) ?? AcquisitionBoard.NotSet.index],
+      SEXKEY: Sex.values[(prefs.getInt(SEXKEY)) ?? Sex.NotSet.index],
+    };
+  }
```

Comment:
This assumes information about settings. I think we should just access settings one by one using the `prefs.get` method

Reply:
Si je stocke les strings des enums au lieu des index ça va revenir au même.

Reply:
J'vois mal en quoi c'est problématique vérifier que tes données persistentes sont cohérentes avec ta logique.